pkg/tcpip/stack/stack.go

```diff
@@ -390,6 +391,10 @@ type Stack struct {
 		mu struct {
 			sync.RWMutex
 
+			// table is in descending sort order based on
+			// the prefix length. Finding a route in the
+			// table will result in Longest prefix match
+			// as is expected.
```

Comment:
I agree with this, but if you can find an RFC quote or something similar to document here?